be/src/runtime/stream_load/stream_load_executor.cpp

```diff
@@ -320,12 +320,16 @@ Status StreamLoadExecutor::commit_txn(StreamLoadContext* ctx) {
     TNetworkAddress master_addr = _exec_env->cluster_info()->master_fe_addr;
     TLoadTxnCommitResult result;
 #ifndef BE_TEST
+    int timeout_ms = config::txn_commit_rpc_timeout_ms;
```

Comment:
should also handle the commit the `group_commit_mgr`

Reply:
@mymeiyi done